package/yast2-ruby-bindings.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Thu Jun 25 12:40:48 UTC 2020 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Backported jreidinger's patches to SLE-12-SP4 (bsc#1172848):
```

Comment:
just nitpick, it is not backport, but forward port, as original patches was for SLE12 SP2